src/core/colorspace.js

```diff
@@ -304,7 +304,7 @@ var ColorSpace = (function ColorSpaceClosure() {
           }
           break;
         case 'Pattern':
-          var basePatternCS = xref.fetchIfRef(cs[1]) || null;
```

Comment:
*Note:* In this case, `xref.fetchIfRef` wasn't actually necessary given https://github.com/mozilla/pdf.js/blob/master/src/core/colorspace.js#L240. The reason that I recently, in PR #6082, added it is simple because I didn't read the code properly.